src/ray/core_worker/transport/scheduling_util.cc

```diff
@@ -59,6 +59,10 @@ DependencyWaiterImpl::DependencyWaiterImpl(DependencyWaiterInterface &dependency
 
 void DependencyWaiterImpl::Wait(const std::vector<rpc::ObjectReference> &dependencies,
                                 std::function<void()> on_dependencies_available) {
+  if (dependencies.size() == 0) {
```

Comment:
This is possible to pass 0 in-plasma dependencies if all deps are in-actor deps.